spotify-app/demiv2/script/main.js

```diff
@@ -0,0 +1,156 @@
+(function() {  

```

Comment:
hey! why you need to instance all your `scopes` in a SEAF ?

Reply:
ummmmm jajajaja sincerely i don't really know at 80% but i saw an example that was like this, if it s not necessary i will change it